pkg/test/auth/tokenmanager_test.go

```diff
@@ -173,3 +173,18 @@ func TestTokenManagerKeyService(t *testing.T) {
 		require.Equal(t, key1.N, rsaKey1.N)
 	})
 }
+
+func TestTokenManagerE2ETestKeys(t *testing.T) {
```

Comment:
In our tests we should:

- get the JWK key(s) using `GetE2ETestKeys()`. Then parse it and convert to an `rsa.PublicKey` key.
For doing that I would move https://github.com/codeready-toolchain/registration-service/blob/master/pkg/auth/keymanager.go#L77-L123 to toolchain-common/tree/master/pkg/auth/keys.go as util functions and reused them here and in reg-service.
- generate a token using the private key (we should add this function to the test manager which will be consumed by e2e tests, see my comments above)
- validate the generated token using the public key you got in the first step

Reply:
we don't actually need it in the rsa.PublicKey format as far as i can see. fetchkeysFromBytes actually returns []*PublicKey. so i changed GetE2ETestPublicKey to return []*PublicKey and now we can use that in replace of fetchKeysFromBytes in registration-service since it returns the correct format

fetchKeysFromBytes: https://github.com/codeready-toolchain/registration-service/blob/master/pkg/auth/keymanager.go#L114